### PR TITLE
add signed links generating, csv file parsing

### PR DESCRIPTION
**Pull request by Gennadiixd**
### DONE: (score 7/8)
5.1. Presigned link generation.
 - CSV file uploading (integrated with FE https://d1qnity9rv2xzx.cloudfront.net/)

5.2. CSV file parsing and logging in console.

### Additional
 - async/await is used in lambda functions
 - At the end of the stream the lambda function should move the file from the uploaded folder into the parsed folder (move the file means that file should be copied into parsed folder, and then deleted from uploaded folder)

### NOT DONE:
 - importProductsFile lambda is covered by unit tests (aws-sdk-mock can be used to mock S3 methods - https://www.npmjs.com/package/aws-sdk-mock)

### CODE REVIEW

File: import-service/handlers/import-products-file.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import "source-map-support/register";
+import S3 from "aws-sdk/clients/s3";
+import { Response } from "./response";
+
+export const importProductsFile = async (event) => {
```
Comment (BohdanRSchool) [inline import-service/handlers/import-products-file.ts]: if you use a getObject operation (synchronously) you don't need async

File: import-service/serverless.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  },
+  functions: {
+    importProductsFile: {
+      handler: "handler.importProductsFile",
+      events: [
+        {
+          http: {
+            method: "get",
+            path: "import",
+            cors: {
```
Comment (BohdanRSchool) [inline import-service/serverless.ts:50]: you can add `request: {  parameters: {   querystrings: {  name: true   } } }`

File: package.json [hunk 3]
```diff
@@ -8,7 +8,12 @@
     "test": "jest"
   },
   "dependencies": {
+    "@types/aws-lambda": "^8.10.64",
```
Comment (BohdanRSchool) [inline package.json:11]: do you need all these packages in the root and import-service package.json?

**Comment by BohdanRSchool** [comment]
did you create the PR for FE repo?

**Review by Savsan — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (Gennadiixd) [reply package.json:11]: Will clean it up later, it need for some dirty tests =)

**Pull request merged**